ding/framework/middleware/ckpt_handler.py

```diff
@@ -23,6 +35,17 @@ def __init__(self, cfg: EasyDict, policy: Policy, train_freq: Optional[int] = No
         self.max_eval_value = -np.inf
 
     def __call__(self, ctx: Union["OnlineRLContext", "OfflineRLContext"]) -> None:
+        """
+        Overview:
+            The method to save checkpoint data. \
+            The checkpoint data will be saved in a file in following 3 cases: \
+                - Every time when train self.train_freq iterations; \
+                - Every time when the eval reward is the best eval reward so far; \
+                - When task.finish is True. \
+        Input of ctx:
+            - train_iter (:obj:`int`): Number of train_iter.
```

Comment:
Number of training iteration, i.e. the number of updating policy related network